crates/containerd-shim-wasm/src/sandbox/instance.rs

```diff
@@ -219,12 +219,9 @@ mod noptests {
     fn test_nop_kill_sigkill() -> Result<(), Error> {
         let nop = Arc::new(Nop::new("".to_string(), None));
```

Comment:
IIUC, you don't need this `Arc` either.
Same for all the other ones in the PR (inc. the `wasi` one).

Reply:
Hmm, I think you are right. Removed the Arc wrapping from tests now.